src/pages/login/login.ts

```diff
@@ -0,0 +1,68 @@
+import { Component } from '@angular/core';
+import {AlertController, LoadingController, NavController, NavParams} from 'ionic-angular';
+
+//@Pages
+import {HomePage} from "../home/home";
+
+//@Models
+import {Credentials} from "../../models/credentials";
+import {AuthServiceProvider} from "../../providers/auth-service/auth-service";
+import {User} from "../../models/user";
+
+
+@Component({
+  selector: 'page-login',
+  templateUrl: 'login.html',
+})
+export class LoginPage {
+
+  loginData : Credentials;
+  usrName   : string;
+  usrPsw    : string;
+  user      : User;
+
+  constructor(public navCtrl: NavController, public navParams: NavParams,
+              public authService : AuthServiceProvider,
+              public loadingCtrl : LoadingController,
+              public alertCtrl   : AlertController) {
+  }
+
+  ionViewDidLoad() {
+    console.log('ionViewDidLoad LoginPage');
+  }
+
+  login() {
+    this.loginData = new Credentials(this.usrName, this.usrPsw);
+    this.user = this.authService.authUser(this.loginData);
+    this.waitForResponse().then(() => {
+      setTimeout(() => {
+
+        if(this.user != null){
```

Comment:
I suggest to change this validation. 
If you use `if(this.user)`, you are validating if the property exist and is not null or undefined or false.